sys/vane/rver.hoon

```diff
@@ -1333,10 +1337,12 @@
       |=  [channel-id=@t json-text=wall]
       ^-  [(list move) server-state]
       ::
-      =/  channel=channel
-        (~(got by session.channel-state.state) channel-id)
+      =/  channel=(unit channel)
+        (~(get by session.channel-state.state) channel-id)
+      ?~  channel
+        [~ state]
```

Comment:
Please add a ~& here; I'm still curious about actually figuring out why we can get into a state where the channel has stopped existing but is still receiving callbacks.

Reply:
Is there anything in particular you want me to print out, or just a term to indicate it got here?

Reply:
Indicating that it got there, though you might as well throw the channel-id in the logging statement. I'm not convinced it can get here after the changes to subscribe/unsubscribe, so traces showing it can would be super interesting.